nitrocli/src/arg_util.rs

```diff
@@ -76,6 +76,8 @@ macro_rules! Command {
         }
       }
     }
+
+    impl_default!($name => $( $name::$var(::std::default::Default::default()) , )*);
```

Comment:
Seems to me a bit like a misuse of `Default`. Or at least a "dangerous" interpretation of it. But I guess it's only temporary as well?

Reply:
I think it’s an acceptable implementation, but it’s only temporary anyway.  For structopt, we have to construct an instance of the subcommand enums.  That would be very complicated without `Default`.  The implementation and the macro are removed in the “Remove unused code after structopt transition“ commit.